api/src/messaging/consumers.py

```diff
@@ -149,24 +72,18 @@ async def send_like(self, event):
             )
 
     async def send_message(self, event):
```

Comment:
Although we cannot move this method into another file, I think we move the logic of creating the data to be send into a separate function. We want the ChatConsumer to have minimal logic, and just serve as the entry point for all of the websocket specific functionality